Library/Formula/docker-machine.rb

```diff
@@ -22,7 +22,9 @@ def install
     mkdir_p buildpath/"src/github.com/docker/"
     ln_sf buildpath, buildpath/"src/github.com/docker/machine"
 
+    ENV["PWD"] = buildpath/"src/github.com/docker/machine/"
```

Comment:
I initially did that like so:

```ruby
Dir.chdir(buildpath/"src/github.com/docker/machine/") do
  system "make", "build"
end
```

But is doesn't work as the destination directory is a symlink to the `buildpath`, meaning the current path looks the same to `make` both inside and outside the block.